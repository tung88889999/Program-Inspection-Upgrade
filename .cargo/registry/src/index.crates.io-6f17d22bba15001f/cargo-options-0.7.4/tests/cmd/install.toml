bin.name = "cargo-mimic"
args = "install --help"
