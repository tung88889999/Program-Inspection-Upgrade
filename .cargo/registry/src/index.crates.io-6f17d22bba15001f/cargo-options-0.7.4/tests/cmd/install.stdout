Install a Rust binary. Default location is $HOME/.cargo/bin

Usage: cargo-mimic[EXE] install [OPTIONS] [crate]...

Arguments:
  [crate]...  

Options:
  -q, --quiet                 Do not print cargo log messages
      --message-format <FMT>  Error format
  -v, --verbose...            Use verbose output (-vv very verbose/build.rs output)
      --color <WHEN>          Coloring: auto, always, never
      --config <KEY=VALUE>    Override a configuration value (unstable)
  -Z <FLAG>                   Unstable (nightly-only) flags to Cargo, see 'cargo -Z help' for
                              details
      --version <VERSION>     Specify a version to install
      --git <URL>             Git URL to install the specified crate from
      --branch <BRANCH>       Branch to use when installing from git
      --tag <TAG>             Tag to use when installing from git
      --rev <SHA>             Specific commit to use when installing from git
      --path <PATH>           Filesystem path to local crate to install
      --list                  list all installed packages and their versions
  -f, --force                 Force overwriting existing crates or binaries
      --no-track              Do not save tracking information
      --debug                 Build in debug mode (with the 'dev' profile) instead of release mode
      --root <DIR>            Directory to install packages into
      --index <INDEX>         Registry index to install from
      --registry <REGISTRY>   Registry to use
  -h, --help                  Print help

Compilation Options:
  -j, --jobs <N>                Number of parallel jobs, defaults to # of CPUs
      --keep-going              Do not abort the build as soon as there is an error (unstable)
      --profile <PROFILE-NAME>  Build artifacts with the specified Cargo profile
      --target <TRIPLE>         Build for the target triple [env: CARGO_BUILD_TARGET=]
      --target-dir <DIRECTORY>  Directory for all generated artifacts
      --timings[=<FMTS>...]     Timing output formats (unstable) (comma separated): html, json

Feature Selection:
  -F, --features <FEATURES>  Space or comma separated list of features to activate
      --all-features         Activate all available features
      --no-default-features  Do not activate the `default` feature

Manifest Options:
      --frozen   Require Cargo.lock and cache are up to date
      --locked   Require Cargo.lock is up to date
      --offline  Run without accessing the network

Target Selection:
      --bin [<NAME>]      Install only the specified binary
      --bins              Install all binaries
      --example [<NAME>]  Install only the specified example
      --examples          Install all examples

Run `cargo help install` for more detailed information.
